src/test/TestObject.cpp

```diff
@@ -20,7 +20,7 @@
 
 #include <sstream>
 
-#include "ILogger.h"
+#include "utility/ILogger.h"
```

Comment:
```suggestion
#include "../utility/ILogger.h"
```

This is fine in a source file, but for the sake of consistency, I'd go with the suggestion.